scripts/bench/extract-component.js

```diff
@@ -112,7 +112,7 @@ function print(outerComponent) {
         var values = keys.map((childKey) => renderedChildren[childKey]);
 
         if (keys.length) {
-          function dump(children) {
+          var dump = function(children) {
```

Comment:
it's better to write `var dump = function dump(children) {` to keep function names in stack trace